app/src/main/java/app/revanced/integrations/youtube/utils/VideoUtils.java

```diff
@@ -130,6 +133,27 @@ public static void launchExternalDownloader(@NonNull String videoId) {
         }
     }
 
+    public static void launchPlaylistExternalDownloader(@NonNull String playlistId) {
+        try {
+            // use default downloader (YTDLnis) if user tries to download a playlist
+            // because most downloaders don't support playlist download
+            String downloaderPackageName = externalDownloaderPackageName.defaultValue;
```

Comment:
why not hardcoded the value there 👀 So if one day, inotia change the default value to Seal, or new downloader, then this feature will still worked as intended